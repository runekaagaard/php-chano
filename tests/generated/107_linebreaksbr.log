
---- EXPECTED OUTPUT
123
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::linebreaksbr() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/107_linebreaksbr.php on line 4
---- FAILED
