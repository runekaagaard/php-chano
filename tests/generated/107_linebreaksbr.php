<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 123)));
foreach ($items as $i) echo $i->input->linebreaksbr();
