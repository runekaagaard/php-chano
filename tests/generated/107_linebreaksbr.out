Fatal error: Call to undefined method DtlIter::linebreaksbr() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/107_linebreaksbr.php on line 4